src/visuallyCompleteCalculator.ts

```diff
@@ -166,13 +205,31 @@ class VisuallyCompleteCalculator {
     this.subscribers.forEach((subscriber) => subscriber(measurement));
   }
 
+  private nextCancellation(cancellation: Cancellation) {
+    Logger.debug(
+      'VisuallyCompleteCalculator.nextCancellation()',
```

Comment:
Any reason not to do:
```
`VisuallyCompleteCalculator.nextCancellation() :: cancellationreason = ${cancellation.reason}`;
```

Reply:
No, no strong reason.  I just found this pattern generalized fairly well to all of the use cases in this repo.